src/rtctools_heat_network/heat_physics_mixin.py

```diff
@@ -1401,15 +1446,46 @@ def __pipe_heat_to_discharge_path_constraints(self, ensemble_member):
                 np.abs((*self.bounds()[f"{p}.HeatIn.Heat"], *self.bounds()[f"{p}.HeatOut.Heat"]))
             )
 
+            carrier = parameters[f"{p}.carrier_id"]
+            temperatures = self.temperature_regimes(carrier)
+
             for heat in [scaled_heat_in, scaled_heat_out]:
                 if self.heat_network_options()["neglect_pipe_heat_losses"]:
-                    constraints.append(
-                        (
-                            (heat - pipe_q * (cp * rho * temp)) / heat_nominal,
-                            0.0,
-                            0.0,
+                    if len(temperatures) == 0:
```

Comment:
Do I see correctly that this is a bug fix? not specific to ATES

Reply:
Indeed, not of high priority so I thought it could just take it along in this development